src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in

```diff
@@ -397,8 +408,9 @@ class MiniBatchStandardizer:
             x_std_dev_str = self.x_std_dev_str)
         return query
 
-    def _get_query_for_standardizing_with_grouping(self):
+    def _create_table_for_standardizing_with_grouping(self):
```

Comment:
Why was the method name changed? The older name seems to be more apt, since this function is still returning the query, and not executing it (the same for `_create_table_for_standardizing_without_grouping()` too).

Reply:
yes you are right. It was initially creating the table inside the function and later I moved it outside but forgot to rename the function. will update the name